### PR TITLE
added or type for display and text-align

### PR DESCRIPTION
**Pull request by MeenuKumari28**
added or type for display and text-align

### CODE REVIEW

File: demos/dropbase.ftd [hunk 1]
```diff
@@ -283,7 +283,7 @@ inner-background.solid: $ds.colors.background.step-1
 inset: $ds.spaces.inset-square.large
 
 -- ds.copy-large:
-text-align: center
+display-align: block
```
Comment (amitu) [inline demos/dropbase.ftd:286]: Not fond of `display-align` name.

File: typography.ftd [hunk 2]
```diff
@@ -1,124 +1,174 @@
+-- or-type display-type:
+
+-- constant integer inline: 1
+-- constant integer block: 2
+-- constant integer inline-block: 3
+-- constant integer text-center: 4
+-- constant integer text-left: 5
+-- constant integer text-right: 6
+-- constant integer text-justify: 7
```
Comment (amitu) [inline typography.ftd:9]: The `text-` prefix use useless, we are applying it on text only.

File: typography.ftd [hunk 3]
```diff
@@ -1,124 +1,174 @@
+-- or-type display-type:
+
+-- constant integer inline: 1
+-- constant integer block: 2
```
Comment (Arpita-Jaiswal) [inline typography.ftd:4]: This variant is not needed. As per your implementation, this does the same thing as `text-center`.

File: demos/dropbase.ftd (hunk 1 above)
Comment (MeenuKumari28) [reply demos/dropbase.ftd:286]: Is `align` a better name? @amitu

Comment (amitu) [reply demos/dropbase.ftd]: Yes.

**Review by amitu — APPROVED** [review]

**Pull request merged**